packages/chaire-lib-backend/src/utils/processManagers/OSRMProcessManager.ts

```diff
@@ -129,7 +137,7 @@ function getOsrmRoutedStartArgs(osrmDirectory: string, mode: string, port: strin
 //TODO set type for Promise return (in all the file)
 const start = function (parameters = {} as any): Promise<any> {
     const mode = parameters.mode || 'walking';
-    const port = parameters.port || Preferences.get(`osrmRouting.${mode}.port`);
+    const port = parameters.port || ServerConfig.getRoutingEngineConfigForMode(mode, 'osrmRouting').port;
```

Comment:
Just wondering if we need this, maybe add a TODO to force receiving the mode and port (because the caller will have checked it already) and not initialize to config here